packages/netlify-cms-widget-select/src/SelectControl.js

```diff
@@ -1,31 +1,52 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ImmutablePropTypes from 'react-immutable-proptypes';
-import { Map } from 'immutable';
+import { Map, List, fromJS } from 'immutable';
 import { find } from 'lodash';
 import Select from 'react-select';
 import { colors } from 'netlify-cms-ui-default';
 
 const styles = {
-  control: provided => ({
-    ...provided,
+  control: styles => ({
+    ...styles,
     border: 0,
     boxShadow: 'none',
     padding: '9px 0 9px 12px',
   }),
-  option: (provided, state) => ({
-    ...provided,
+  option: (styles, state) => ({
+    ...styles,
     backgroundColor: state.isSelected
       ? `${colors.active}`
       : state.isFocused
         ? `${colors.activeBackground}`
         : 'transparent',
     paddingLeft: '22px',
   }),
-  menu: provided => ({ ...provided, right: 0 }),
-  container: provided => ({ ...provided, padding: '0 !important' }),
-  indicatorSeparator: () => ({ display: 'none' }),
-  dropdownIndicator: provided => ({ ...provided, color: `${colors.controlLabel}` }),
+  menu: styles => ({ ...styles, right: 0 }),
+  container: styles => ({ ...styles, padding: '0 !important' }),
+  indicatorSeparator: (styles, state) =>
+    state.hasValue && state.selectProps.isClearable
+      ? { ...styles, backgroundColor: `${colors.textFieldBorder}` }
+      : { display: 'none' },
+  dropdownIndicator: styles => ({ ...styles, color: `${colors.controlLabel}` }),
+  clearIndicator: styles => ({ ...styles, color: `${colors.controlLabel}` }),
+  multiValue: styles => ({
+    ...styles,
+    backgroundColor: colors.background,
+  }),
+  multiValueLabel: styles => ({
+    ...styles,
+    color: colors.textLead,
+    fontWeight: 500,
+  }),
+  multiValueRemove: styles => ({
+    ...styles,
+    color: colors.controlLabel,
+    [':hover']: {
```

Comment:
you can drop the brackets here 😄 

```suggestion
    ':hover': {
```

Reply:
Ah nice catch 😄